react-state/my-app/src/Image.tsx

```diff
@@ -0,0 +1,28 @@
+import './Image.css';
+import { useState } from 'react';
+
+type Props = {
+  srcs: string[];
+};
+export function Image({ srcs }: Props) {
+  const [currentImage, setCurrentImage] = useState(0);
```

Comment:
Not a big deal, but we can use a better variable name. The value is not the current image, but its index in the srcs array. "currImageIndex", "imgIdx", "imageIndex", etc.